src/status_im/data_store/switcher_cards.cljs

```diff
@@ -0,0 +1,12 @@
+(ns status-im.data-store.switcher-cards
+  (:require [utils.re-frame :as rf]
+            [taoensso.timbre :as log]))
+
+(rf/defn fetch-switcher-cards-rpc
+  [_]
+  {:json-rpc/call [{:method     "wakuext_switcherCards"
```

Comment:
just wondering why this one in a separate ns ?

Reply:
fixed, thank you